data/variants.txt

```diff
@@ -1265,6 +1265,434 @@ ship "Heavy Shuttle" "Heavy Shuttle (Armed)"
+	add attributes
+		"shields" 3000
+		"hull" 1500
+	outfits
+		Hyperdrive
+		"Heavy Anti-Missile Turret" 4
+		"A525 Atomic Steering"
+		"A125 Atomic Steering"
+		"A860 Atomic Thruster"
+		"Liquid Nitrogen Cooler"
+		"Torpedo Launcher" 6
+		"Tactical Scanner" 2
+		Torpedo 270
+		"LP144a Battery Pack"
+		"Armageddon Core"
+		"S-970 Regenerator"
+		"Torpedo Storage Rack" 6
+		"Laser Rifle" 128
+		
+		
+		
+ship "Kestrel" "KestrelDef (Hai)"
+	add attributes
+		"shields" 3000
+		"hull" 1500
+	outfits
+		"Outfits Expansion" 2
+		Hyperdrive
+		"Hai Ravine Batteries"
+		"Liquid Helium Cooler"
+		"Quantum Keystone"
+		"Hai Diamond Regenerator"
+		"Boulder Reactor"
+		"A865 Atomic Steering"
+		"Pulse Cannon" 2
+		"Tracker Storage Pod"
+		"Ion Cannon" 2
+		"Hai Tracker" 140
+		"Hai Tracker Pod" 2
+		"Pulse Turret" 4
+		`"Bufaer" Atomic Thruster`
+		"Geode Reactor"
+		`"Basrem" Atomic Steering`
+		"Tactical Scanner" 2
+		"Pulse Rifle" 128
+		
+		
+		
+ship "Kestrel" "KestrelWep (Particle)"
+	add attributes
+		"weapon capacity" 40
+	outfits
+		"Outfits Expansion" 6
+		Hyperdrive
+		"Liquid Helium Cooler"
+		"A255 Atomic Steering"
+		"A865 Atomic Steering"
+		"A520 Atomic Thruster"
+		"Particle Cannon" 6
+		"Liquid Nitrogen Cooler"
+		"Torpedo Launcher" 4
+		Torpedo 120
+		"Armageddon Core"
+		"S-970 Regenerator"
+		"Heavy Laser" 6
+		"LP288a Battery Pack"
+		"Laser Rifle" 128
+	gun -31 66 "Heavy Laser"
+	gun 31 66 "Heavy Laser"
+	gun -31 66 "Particle Cannon"
+	gun 31 66 "Particle Cannon"
+	gun -31 66 "Particle Cannon"
+	gun 31 66 "Particle Cannon"
+	gun -53 61 "Particle Cannon"
+	gun 53 61 "Particle Cannon"
+	gun -53 61 "Torpedo Launcher"
+	gun 53 61 "Torpedo Launcher"
+	gun -53 61 "Heavy Laser"
+	gun 53 61 "Heavy Laser"
+	gun -75 57 "Torpedo Launcher"
+	gun 75 57 "Torpedo Launcher"
+	gun -75 57 "Heavy Laser"
+	gun 75 57 "Heavy Laser"
+	gun -75 57
+	gun 75 57
+	
+	
+	
+ship "Kestrel" "KestrelWep (Laser)"
+	add attributes
+		"weapon capacity" 40
+	outfits
+		"Outfits Expansion" 4
+		Hyperdrive
+		"Liquid Helium Cooler"
+		"Electron Beam" 12
+		"A865 Atomic Steering"
+		"A250 Atomic Thruster"
+		"A520 Atomic Thruster"
+		"Tactical Scanner" 2
+		"Armageddon Core"
+		"S-970 Regenerator"
+		"Heavy Laser" 6
+		"LP288a Battery Pack"
+		"Laser Rifle" 128
+	gun -31 66 "Heavy Laser"
+	gun 31 66 "Heavy Laser"
+	gun -31 66 "Heavy Laser"
+	gun 31 66 "Heavy Laser"
+	gun -31 66 "Heavy Laser"
+	gun 31 66 "Heavy Laser"
+	gun -53 61 "Electron Beam"
+	gun 53 61 "Electron Beam"
+	gun -53 61 "Electron Beam"
+	gun 53 61 "Electron Beam"
+	gun -53 61 "Electron Beam"
+	gun 53 61 "Electron Beam"
+	gun -75 57 "Electron Beam"
+	gun 75 57 "Electron Beam"
+	gun -75 57 "Electron Beam"
+	gun 75 57 "Electron Beam"
+	gun -75 57 "Electron Beam"
+	gun 75 57 "Electron Beam"
+	
+	
+	
+ship "Kestrel" "KestrelWep (Plasma)"
+	add attributes
+		"weapon capacity" 40
+	outfits
+		"Outfits Expansion" 6
+		Hyperdrive
+		"Liquid Helium Cooler" 2
+		"A255 Atomic Steering"
+		"Electron Beam" 12
```

Comment:
This variant can appear before the start of the campaign.

Reply:
Did you mean to type "can't"?

I'm going to assume you did :D This is a tricky one. You can already run into Electron Marauders prior to FW beginning, already in the base game. I felt that this was a bit unusual but that's how vanilla is, so I figured it would be ok to add similarly fit Kestrels.

Reply:
I definitely mean can, along with shouldn't (outside marauders). In `event "kestrel available: more weapons" there are three non marauder fleet variants fleet (`"Large Southern Pirates"``fleet "Large Northern Merchants"``fleet "Large Deep Merchants"`) contain this variant, this event can trigger before the start of the campaign.

Reply:
All Kestrel fits that appear in Merchant/Pirate fleets have been reworked to not include Syndicate shields or Electron weapons. I think.